x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_8.0.0/mappings.json

```diff
@@ -31,7 +31,7 @@
           "core-usage-stats": "3d1b76c39bfb2cc8296b024d73854724",
           "dashboard": "40554caf09725935e2c02e02563a2d07",
           "endpoint:user-artifact": "4a11183eee21e6fbad864f7a30b39ad0",
-          "endpoint:user-artifact-manifest": "4b9c0e7cfaf86d82a7ee9ed68065e50d",
+          "endpoint:user-artifact-manifest": "a0d7b04ad405eed54d76e279c3727862",
```

Comment:
Anyone know why these non-Security Solution test suites were impacted?

Reply:
I'm assuming (maybe) that they include es archives of Saved Objects and (accidently??) includes our types?

Reply:
I'm not sure, but it's all over the place.